docs/index.md

```diff
@@ -21,6 +21,34 @@ The goal is to provide a smooth out-of-box experience for simple use cases, and
 allow swapping in more powerful backends, like Mesos, for large scale production
 deployments.
 
+## Release Notes
+
+### Version 0.2.0 (April 7, 2015)
```

Comment:
Yes, I get that. These are release notes for that version, for when it is released.